org.godotengine.Godot3.yaml

```diff
@@ -0,0 +1,95 @@
+app-id: org.godotengine.Godot3
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: godot
+
+build-options:
+  arch:
+    x86_64:
+      env:
+        # Only enable link-time optimization when targeting x86_64
+        # (causes issues on other architectures)
+        SCONS_FLAGS_EXTRA: use_lto=yes
+
+  env:
+    # Will be appended to the version string displayed in the editor and command-line help
+    BUILD_NAME: flathub
+
+    # SCons flags common to all builds
+    # Shouldn't be quoted when used as it's a single string, not an array
+    SCONS_FLAGS: >
+      platform=x11
+      CCFLAGS=-I/app/include
+      prefix=/app
+      unix_global_settings_path=/app
+      progress=no
+      builtin_freetype=no
+      builtin_libogg=no
+      builtin_libpng=no
+      builtin_libtheora=no
+      builtin_libvorbis=no
+      builtin_libwebp=no
+      builtin_openssl=no
+      builtin_libvpx=no
+      builtin_zlib=no
+      udev=no
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --device=all
+
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: scons
+    buildsystem: simple
+    cleanup: ['*']
+
+    sources:
+      - type: archive
+        sha256: 3d43b2303a924816ea0e1b345ff04c9b3e27b53eadf0f26012fc0c29b019685f
+        url: https://downloads.sourceforge.net/project/scons/scons/4.4.0/SCons-4.4.0.tar.gz
+
+    build-commands:
+      - pip3 install --no-index --no-build-isolation --prefix=/app .
+
+  - name: godot-tools
+    buildsystem: simple
+
+    sources:
+      - type: archive
+        sha256: b43a1d6b29ea25bcbafbcebf7ba0843261aa1237e2739494538a2200d8c7be62
+        url: https://downloads.tuxfamily.org/godotengine/3.5.2/godot-3.5.2-stable.tar.xz
+
+      - type: script
+        dest-filename: godot.sh
+        commands:
+          - export APPDATA="$XDG_DATA_HOME"
+          - export PATH="/app/jre/bin:$PATH"
+          - /app/bin/godot-bin "$@"
+
+      - type: file
+        path: org.godotengine.Godot3.appdata.xml
+
+    build-commands:
+      - python3 /app/bin/scons $SCONS_FLAGS $SCONS_FLAGS_EXTRA tools=yes target=release_debug -j "$FLATPAK_BUILDER_N_JOBS"
+      - install -D -m755 bin/godot.x11.opt.tools.* /app/bin/godot-bin
+      - install -D -m755 godot.sh /app/bin/godot
+      - mv misc/dist/linux/org.godotengine.Godot.desktop misc/dist/linux/$FLATPAK_ID.desktop
+      - mv misc/dist/linux/org.godotengine.Godot.xml misc/dist/linux/$FLATPAK_ID.xml
```

Comment:
you'll need to change the app-id in the appdata file.

Reply:
also there are duplicates. line 89 and 92. lin 90 and 94.

Reply:
> you'll need to change the app-id in the appdata file.

You mean to tell me _that_ was why it wasn't working? :man_facepalming: